pkg/registry/core/service/ipallocator/controller/repair.go

```diff
@@ -74,6 +74,7 @@ const numRepairsBeforeLeakCleanup = 3
 // and generates informational warnings for a cluster that is not in sync.
 func NewRepair(interval time.Duration, serviceClient corev1client.ServicesGetter, eventClient eventsv1client.EventsV1Interface, network *net.IPNet, alloc rangeallocation.RangeRegistry, secondaryNetwork *net.IPNet, secondaryAlloc rangeallocation.RangeRegistry) *Repair {
 	eventBroadcaster := events.NewBroadcaster(&events.EventSinkImpl{Interface: eventClient})
+	defer eventBroadcaster.Shutdown()
```

Comment:
@wojtek-t was working on this controller, and modifying its lifecyle, we should better not modify it here to avoid conflicts

Reply:
https://github.com/kubernetes/kubernetes/pull/109975 is the PR (although there is some issue with it now)

Reply:
But +1 for removing it from this PR.